src/lib/components/input/Input.svelte

```diff
@@ -1,79 +1,94 @@
 <script lang="ts">
-	import { fly } from 'svelte/transition'
-	import type { HTMLInputTypeAttribute } from '../../../types/html-input-type-attribute.type'
+  import { fly } from 'svelte/transition'
+  import type { HTMLInputTypeAttribute } from '../../../types/html-input-type-attribute.type'
```

Comment:
```suggestion
  import { fly } from 'svelte/transition'
  
  import type { HTMLInputTypeAttribute } from '$lib/types/html-input-type-attribute.type'
```
El espacio para mantener el estilo de imports del mismo tipo (de módulos del lenguaje, de módulos third party y de módulos propios) 
Lo segundo implica mover el html-input-type-attribute a types en lib

Reply:
arreglao